firebase-config/src/main/java/com/google/firebase/remoteconfig/RemoteConfigComponent.java

```diff
@@ -199,9 +207,11 @@ synchronized FirebaseRemoteConfig get(
               defaultsClient,
               fetchHandler,
               getHandler,
-              metadataClient);
+              metadataClient,
+              getRealtime(firebaseApp, firebaseInstallations, fetchHandler, context, namespace));
       in.startLoadingConfigsFromDisk();
       frcNamespaceInstances.put(namespace, in);
+      frcNamespaceInstancesBackground.put(namespace, in);
```

Comment:
All instances are both in `frcNamespaceInstances` and `frcNamespaceInstancesBackground`. Why do you need both maps?

Reply:
Originally the background listener was static so I had a static map. Changed to use frcNamespaceInstances.